WordPress/Classes/System/3DTouch/WP3DTouchShortcutHandler.swift

```diff
@@ -0,0 +1,70 @@
+import UIKit
+
+public class WP3DTouchShortcutHandler: NSObject
+{
+    enum ShortcutIdentifier: String {
+        case LogIn
+        case NewPost
+        case NewPhotoPost
+        case Stats
+        case Notifications
+        
+        init?(fullType: String) {
+            guard let last = fullType.componentsSeparatedByString(".").last else { return nil }
```

Comment:
(Maybe) the `{ }` block "not embedded into a single line" ? (Just for the sake of easier lecture)

Not sure, though, since we don't really have a solid standard!

```
else {
   return nil
}
```